From 205f50c3754605c671666ec7e2740c12f2185f55 Mon Sep 17 00:00:00 2001
From: StanleyYP Wang <StanleyYP.Wang@mediatek.com>
Date: Mon, 17 Feb 2025 15:55:52 +0800
Subject: [PATCH 71/92] mtk: mt76: mt7996: add 5T5R support for testmode RX

add 5T5R support for testmode RX (especially for mt7992 BE7200 ifem)

Signed-off-by: StanleyYP Wang <StanleyYP.Wang@mediatek.com>
---
 mt7996/mt7996.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/mt7996/mt7996.h b/mt7996/mt7996.h
index 0f2e338a..35a1c5e6 100644
--- a/mt7996/mt7996.h
+++ b/mt7996/mt7996.h
@@ -745,10 +745,10 @@ struct mt7996_phy {
 		u32 *reg_backup;
 
 		s32 last_freq_offset;
-		u8 last_rcpi[4];
-		s8 last_rssi[4];
-		s8 last_ib_rssi[4];
-		s8 last_wb_rssi[4];
+		u8 last_rcpi[5];
+		s8 last_rssi[5];
+		s8 last_ib_rssi[5];
+		s8 last_wb_rssi[5];
 		u8 last_snr;
 
 		u8 spe_idx;
-- 
2.45.2

